modules/ROOT/pages/_partials/upgrade-tool.adoc

```diff
@@ -0,0 +1,61 @@
+//This content is reused in:
+//mule-upgrade-tool.adoc
+
+// Before You Begin
+// tag::BeforeYouBegin[]
+
+* A currently operational Mule instance running version 4.1.1 or later, in _stopped_ status to prepare for the upgrade.
++
+When updating your current Mule instance to a version between 4.1.1 and 4.1.4, use any patch update released after January 20, 2022. If the running version is not one of the previously mentioned, an error message appears stating that the Mule version cannot be upgraded unless an upgrade is made.
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory, that is, your old or existing Mule instance. 
++
+To download the tool, log in to https://help.mulesoft.com/s/[Help Center^]. Under the Support menu, select *Downloads > Mule Runtime*.
++
+An example of the file name is `mule-runtime-upgrade-tool-1.1.0.zip`.
+* The latest Mule runtime engine distribution, downloaded and unpacked in your system.
++
+The Mule upgrade tool requires the full distribution of the latest Mule runtime minor or patch version. Ensure you download a full Mule runtime distribution, identified by including the term `distribution` (and not the term `patch`) in the file name, for example, for the full distribution of the 20220221 patch for Mule 4.4.0: `mule-ee-distribution-standalone-4.4.0-20220221.zip`.
```

Comment:
Do we want to go ahead and use 4.5.0 in the example. Not necessary, but  since this is a 4.5.0 doc, perhaps it makes sense.

Reply:
I would leave the 4.4.0 example as it will appear under the Mule 4.4 branch. We can then modify the example in the 4.5 branch if needed.